src/test/ui/const-generics/issues/issue-62187-encountered-polymorphic-const.min.stderr

```diff
@@ -0,0 +1,10 @@
+warning: unused variable: `foo`
+  --> $DIR/issue-62187-encountered-polymorphic-const.rs:17:9
+   |
+LL |     let foo = <[u8; 2]>::BIT_LEN;
```

Comment:
hmm, while you are at it, can you replace `foo` with `_foo` so we don't emit any warning here?

That would probably be cleaner